doc/whats_new/v0.24.rst

```diff
@@ -256,6 +256,18 @@ Changelog
   :pr:`18280` by :user:`Alex Liang <tianchuliang>` and
   `Guillaume Lemaitre`_.
 
+- |Enhancement| :func:`datasets.fetch_openml` now supports categories with
+  missing values when returning a pandas dataframe. :pr:`18383` by
+  `Thomas Fan`_.
+
+- |Feature| :func:`datasets.fetch_openml` now validates md5checksum of arff
+  files downloaded or cached to ensure data integrity.
+  :pr:`14800` by :user:`Shashank Singh <shashanksingh28>` and `Joel Nothman`_.
+
+- |API| The default value of `as_frame` in :func:`datasets.fetch_openml` is
```

Comment:
This entry was already part of the previous release. You can remove it.

Reply:
done